src/main/java/org/apache/commons/csv/ExtendedBufferedReader.java

```diff
@@ -118,11 +137,53 @@ public int read() throws IOException {
             current == EOF && lastChar != CR && lastChar != LF && lastChar != EOF) {
             lineNumber++;
         }
+        if (encoder != null) {
```

Comment:
Test is missing `enableByteTracking`.

Reply:
`enableByteTracking` will take control on `encoder`, so no extra testing is needed